demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -34,6 +40,134 @@ def __init__(self, release_notes_file_path, modified_files=None, pack_name=None,
         self.release_notes_path = get_release_notes_file_path(self.release_notes_file_path)
         self.latest_release_notes = get_latest_release_notes_text(self.release_notes_path)
 
+    def filter_nones(self, ls: Union[List, Tuple]) -> List:
+        """
+            Filters out None values from a list or tuple.
+        Args:
+            ls: (List | Tuple) - This list or tuple to filter.
+        Return:
+            List filtered from None values.
+        """
+        return list(filter(lambda x: x, ls))
+
+    def extract_rn_headers(self) -> Dict:
+        """
+            Extracts the headers from the release notes file.
+        Args:
+            None.
+        Return:
+            The headers of the release notes file.
+        """
+        headers: Dict = {}
+        # Get all sections from the release notes using regex
+        rn_sections = ENTITY_TYPE_SECTION_REGEX.findall(self.latest_release_notes)
+
+        for section in rn_sections:
+            section = self.filter_nones(ls=section)
+            content_type = section[0]
+            content_type_sections_str = section[1]
+
+            # Extract the sections within a content type. (e.g. Integrations 1,2,3)
+            content_type_sections_ls = ENTITY_SECTION_REGEX.findall(content_type_sections_str)
+            if not content_type_sections_ls:
+                #  Will raise error in validate_special_forms
+                headers[content_type] = []
+            for content_type_section in content_type_sections_ls:
+                content_type_section = self.filter_nones(ls=content_type_section)
+                if content_type_section:
+                    header = content_type_section[0]
+                    if headers.get(content_type):
+                        headers[content_type].append(header)
+                    else:
+                        headers[content_type] = [header]
+        return headers
+
```

Comment:
Please add the '@error_codes' decorator here for the respective validation in the function below.